Makefile

```diff
@@ -538,7 +548,8 @@ st: dist/calicoctl busybox.tar calico-node.tar workload.tar run-etcd calico_test
 ###############################################################################
 .PHONY: ci
 ## Run what CI runs
-ci: static-checks fv image-all st
+# ci: static-checks fv image-all st
+ci: 
```

Comment:
This is debugging right now. Will be reverted once everything with `make k8s-test` is resolved.